sklearn/preprocessing/_encoders.py

```diff
@@ -218,6 +218,18 @@ class OneHotEncoder(_BaseEncoder):
             The ``n_values_`` attribute was deprecated in version
             0.20 and will be removed in 0.22.
 
+    handle_missing : all-missing, all-zero or category
+        What should be done to missing values. Should be one of:
+
+        all-missing: Replace with a row of NaNs as above
```

Comment:
For better rendering, use  restructured text definition list:
```
all-missing
    Replace with a row of NaNs as above
all-zero
    Replace with a row of zeros
```
What do you mean by "as above"

Reply:
Fixed in the last commit.

`as above` is a typo.